cdqa/utils/converters.py

```diff
@@ -135,12 +135,39 @@ def pdf_converter(directory_path):
             df.loc[i] = [pdf, None]
             raw = parser.from_file(os.path.join(directory_path, pdf))
             s = raw["content"]
-            paragraphs = re.split(u"\n(?=\u2028|[A-Z-0-9])", s)
+            paragraphs = re.split("\n(?=\u2028|[A-Z-0-9])", s)
             list_par = []
-            for p in paragraphs:
-                if len(p) >= 200:
-                    list_par.append(p.replace("\n", ""))
-                df.loc[i, "paragraphs"] = list_par
+            line_para = ""
+            if line_paragraph:
+                for p in paragraphs:
+                    if len(p) >= 200:
```

Comment:
You can remove this comparison flow, after some discussions I think we should include smaller paragraphs to the list

Reply:
Hey @andrelmfarias , I have kept the comparison flow for when include_line_breaks is True.
It helps put a limit between paragraphs and line. I am open to suggestions on this, though.

Reply:
Ok for me!

Reply:
Maybe creating a parameter in the function (`max_length` for example) for this with a default value of 200  would be better.

Reply:
Hey @andrelmfarias !  I have added `min_length` parameter in the function as it is the minimum character length to be considered as a paragraph.
Do you think it makes sense?